packages/oui-list/_mixins.less

```diff
@@ -197,7 +197,7 @@
       border-bottom: @separator-size solid @separator-color;
 
       &:last-child {
-          border-bottom: 0 none;
+        border-bottom: 0 none;
```

Comment:
Can be done by stylelint with this plugin: https://github.com/timothyneiljohnson/stylelint-value-border-zero